service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFeedbackReceivedHandler.java

```diff
@@ -80,17 +94,30 @@ public AmqpFeedbackReceivedHandler(String hostName, String userName, String sasT
      * @param sslContext the SSL context to use during the TLS handshake when opening the connection. If null, a default
      *                   SSL context will be generated. This default SSLContext trusts the IoT Hub public certificates.
      */
-    public AmqpFeedbackReceivedHandler(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol, AmqpFeedbackReceivedEvent amqpFeedbackReceivedEvent, ProxyOptions proxyOptions, SSLContext sslContext)
+    public AmqpFeedbackReceivedHandler(
+            String hostName,
+            String userName,
+            String sasToken,
+            IotHubServiceClientProtocol iotHubServiceClientProtocol,
+            AmqpFeedbackReceivedEvent amqpFeedbackReceivedEvent,
+            ProxyOptions proxyOptions,
+            SSLContext sslContext)
     {
         super(hostName, userName, sasToken, iotHubServiceClientProtocol, proxyOptions, sslContext);
-
         this.amqpFeedbackReceivedEvent = amqpFeedbackReceivedEvent;
+    }
 
-        // Add a child handler that performs some default handshaking
-        // behaviour.
-
-        add(new Handshaker());
-        add(new FlowController());
+    AmqpFeedbackReceivedHandler(
```

Comment:
do we need to specify access modifier here? what's the default in Java?

Reply:
No need for one, no. The default is package-private, so this constructor can be called from the one class who instantiates it, but not by users